core-java-modules/core-java-14/pom.xml

```diff
@@ -45,8 +59,9 @@
 	</build>
 
 	<properties>
-		<maven.compiler.source.version>14</maven.compiler.source.version>
-		<maven.compiler.target.version>14</maven.compiler.target.version>
+		<maven.compiler.release>14</maven.compiler.release>
+		<assertj.version>3.6.1</assertj.version>
+        <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
```

Comment:
Couple more lines missing indentation here.

Reply:
Don't know what went wrong here, but let me fix it.